cdap-master-spi/src/main/java/io/cdap/cdap/master/spi/twill/ContainerRuntime.java

```diff
@@ -16,10 +16,17 @@
 
 package io.cdap.cdap.master.spi.twill;
 
+import org.apache.twill.api.TwillRunnable;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
 /**
- * Constants used for twill.
+ * Annotation to denote {@link TwillRunnable} can run on all supported container runtime environments.
  */
-public class TwillConstants {
-  // Configuration used to determine whether to use V1Job to launch twill runnables.
-  public static final String PROGRAM_RUNTIME_ENV = "program.runtime.env";
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE })
+public @interface ContainerRuntime {
```

Comment:
Calling it `ContainerRuntime` to represent runnables that can run on any container runtime supported by CDAP. 
Also open to any other suggestions on the name.